src/main/java/org/apache/sling/feature/launcher/impl/Main.java

```diff
@@ -264,20 +270,23 @@ protected static void parseArgs(final LauncherConfig config, final String[] args
                 config.getConfigClashOverrides().put(name, cfgCProps.getProperty(name));
             }
 
-            Properties fwProps = cl.getOptionProperties(OPT_FRAMEWORK_PROPERTIES);
-            for (final String name : fwProps.stringPropertyNames()) {
-                config.getInstallation()
-                        .getFrameworkProperties()
-                        .put(name, fwProps.getProperty(name));
-            }
+            extractValuesFromOption(cl, OPT_FRAMEWORK_PROPERTIES).orElseGet(ArrayList::new)
+                    .forEach(value -> {
+                        final String[] keyVal = split(value);
+                        config.getInstallation().getFrameworkProperties().put(keyVal[0], keyVal[1]);
+                    });
 
             Properties varProps = cl.getOptionProperties(OPT_VARIABLE_VALUES);
             for (final String name : varProps.stringPropertyNames()) {
                 config.getVariables().put(name, varProps.getProperty(name));
             }
 
-            extractValueFromOption(cl, OPT_VERBOSE, "debug").ifPresent(
-                    value -> System.setProperty("org.slf4j.simpleLogger.defaultLogLevel", value));
+            if (cl.hasOption(OPT_VERBOSE)) {
+                extractValueFromOption(cl, OPT_VERBOSE, "debug").ifPresent(value -> System
+                        .setProperty("org.slf4j.simpleLogger.defaultLogLevel", value));
+            } else {
+                System.setProperty("org.slf4j.simpleLogger.defaultLogLevel", "info");
```

Comment:
I believe that "info" is already the default log level for the simplelogger when there is no "org.slf4j.simpleLogger.defaultLogLevel" system property, so this else block shouldn't be necessary.

Reply:
Ok will change it and cover it up with a test

Reply:
fixed